### DISCUSSION TITLE
Radeon Pro VII - can't overclock

### DISCUSSION THREAD

**Discussion by valeriob01** [preda/gpuowl#222] [Show and tell]
The Radeon Pro VII performance is in all similar to an overclocked Radeon VII, but it can't itself be overclocked due to restrictions.
I expect to see performance gains with a gpuowl version optimized for ROCm 4.x.y, along with performance gains associated with future ROCm 4.x.y version (as stated by @ROCmSupport in https://github.com/RadeonOpenCompute/ROCm/issues/1196#issuecomment-805477176).

Tipical sensor values during gpuowl run (from rocm-smi):
- Temp: 86.0C
- AvgPwr: 186.0W
- SCLK: 1316MHz
- MCLK: 1000MHz
- Fan: 40.0%
- PwrCap: 190.0W
- VRAM%: 2%
- GPU%: 100%

Thus generally the Radeon Pro VII consumes about 50-60 Watts less than a Radeon VII with similar performance.

**Comment by selroc** [comment]
Hi Preda, so you found a way to overclock the Radeon Pro VII, that's good, I will try it after reinstalling my Radeon Pro VII.

**Comment by selroc** [comment]
Attempted to use the upp script to alter Radeon Pro VII performance. I t works, but after a while the system hangs and a hard reset is necessary. Probably 2000 for memory speed is a bit too much for the Radeon Pro VII.
